virttest/utils_time.py

```diff
@@ -14,10 +14,11 @@ def get_host_timezone():
     """
     Get host's timezone
     """
-    timezone_cmd = 'timedatectl | grep "Time zone"'
+    timezone_cmd = 'timedatectl | /usr/bin/grep "Time zone"'
```

Comment:
This should be unnecessary.

Reply:
Thanks. Command `grep --color` may have alias name `grep` in some systems, so get explicit `grep` command with path.

Reply:
Thanks Qianqian, as we talked offline, this is mean to get explicit command 'grep' despite of the alias name.